---------------------
PatchSet 322 
Date: 2010/09/25 09:34:33
Author: millert
Branch: HEAD
Tag: (none) 
Log:
Fix test failures caused by RCS keyword expansion.
Add PERL_GIT_UNCOMMITTED_CHANGES back to git_version.h to fix a test.

Members: 
	git_version.h:1.3->1.4 
	cpan/ExtUtils-MakeMaker/t/parse_version.t:1.2->1.3 
	cpan/Module-Build/t/metadata.t:1.2->1.3 
	cpan/Module-Build/t/moduleinfo.t:1.2->1.3 

Index: src/gnu/usr.bin/perl/git_version.h
diff -u src/gnu/usr.bin/perl/git_version.h:1.3 src/gnu/usr.bin/perl/git_version.h:1.4
--- git_version.h:1.3	Sat Sep 25 13:13:55 2010
+++ git_version.h	Sat Sep 25 16:34:33 2010
@@ -2,6 +2,7 @@
 * WARNING: 'git_version.h' is automatically generated by make_patchnum.pl
 *          DO NOT EDIT DIRECTLY - edit make_patchnum.pl instead
 ***************************************************************************/
+#define PERL_GIT_UNCOMMITTED_CHANGES 
 #define PERL_PATCHNUM ""
 #define PERL_GIT_UNPUSHED_COMMITS		\
     /*leave-this-comment*/
Index: src/gnu/usr.bin/perl/cpan/ExtUtils-MakeMaker/t/parse_version.t
diff -u src/gnu/usr.bin/perl/cpan/ExtUtils-MakeMaker/t/parse_version.t:1.2 src/gnu/usr.bin/perl/cpan/ExtUtils-MakeMaker/t/parse_version.t:1.3
--- cpan/ExtUtils-MakeMaker/t/parse_version.t:1.2	Fri Sep 24 15:06:50 2010
+++ cpan/ExtUtils-MakeMaker/t/parse_version.t	Sat Sep 25 16:34:33 2010
@@ -12,7 +12,7 @@
 
 my %versions = (q[$VERSION = '1.00']            => '1.00',
                 q[*VERSION = \'1.01']           => '1.01',
-                q[($VERSION) = q$Revision: 32208 $ =~ /(\d+)/g;] => 32208,
+                q[($VERSION) = q/Revision: 32208 / =~ /(\d+)/g;] => 32208,
                 q[$FOO::VERSION = '1.10';]      => '1.10',
                 q[*FOO::VERSION = \'1.11';]     => '1.11',
                 '$VERSION = 0.02'               => 0.02,
@@ -34,8 +34,8 @@
                 qq[\$Something::VERSION == 1.0\n\$VERSION = 2.3\n]                     => '2.3',
                 qq[\$Something::VERSION == 1.0\n\$VERSION = 2.3\n\$VERSION = 4.5\n]    => '2.3',
 
-                '$VERSION = sprintf("%d.%03d", q$Revision: 3.74 $ =~ /(\d+)\.(\d+)/);' => '3.074',
-                '$VERSION = substr(q$Revision: 2.8 $, 10) + 2 . "";'                   => '4.8',
+                '$VERSION = sprintf("%d.%03d", q/Revision: 3.74 / =~ /(\d+)\.(\d+)/);' => '3.074',
+                '$VERSION = substr(q/Revision: 2.8 /, 10) + 2 . "";'                   => '4.8',
 
                );
 
Index: src/gnu/usr.bin/perl/cpan/Module-Build/t/metadata.t
diff -u src/gnu/usr.bin/perl/cpan/Module-Build/t/metadata.t:1.2 src/gnu/usr.bin/perl/cpan/Module-Build/t/metadata.t:1.3
--- cpan/Module-Build/t/metadata.t:1.2	Fri Sep 24 15:06:53 2010
+++ cpan/Module-Build/t/metadata.t	Sat Sep 25 16:34:33 2010
@@ -171,9 +171,9 @@
 {
   $dist->change_file( 'lib/Simple.pm', <<'---' );
 package Simple;
-$VERSION = version->new('0.60.' . (qw$Revision: 128 $)[1]);
+$VERSION = version->new('0.60.' . (qw/Revision: 128 /)[1]);
 package Simple::Simon;
-$VERSION = version->new('0.61.' . (qw$Revision: 129 $)[1]);
+$VERSION = version->new('0.61.' . (qw/Revision: 129 /)[1]);
 ---
   $dist->regen;
   my $provides = new_build()->get_metadata()->{provides};
Index: src/gnu/usr.bin/perl/cpan/Module-Build/t/moduleinfo.t
diff -u src/gnu/usr.bin/perl/cpan/Module-Build/t/moduleinfo.t:1.2 src/gnu/usr.bin/perl/cpan/Module-Build/t/moduleinfo.t:1.3
--- cpan/Module-Build/t/moduleinfo.t:1.2	Fri Sep 24 15:06:53 2010
+++ cpan/Module-Build/t/moduleinfo.t	Sat Sep 25 16:34:33 2010
@@ -424,9 +424,9 @@
   # Make sure we handle version.pm $VERSIONs well
   $dist->change_file( 'lib/Simple.pm', <<'---' );
 package Simple;
-$VERSION = version->new('0.60.' . (qw$Revision: 128 $)[1]);
+$VERSION = version->new('0.60.' . (qw/Revision: 128 /)[1]);
 package Simple::Simon;
-$VERSION = version->new('0.61.' . (qw$Revision: 129 $)[1]);
+$VERSION = version->new('0.61.' . (qw/Revision: 129 /)[1]);
 ---
   $dist->regen;
 
